run_acceptance_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+cd /app
+
+pipenv install --dev
+
+OUTPUT=$(pipenv run python generate_qid_batch.py test_batch.csv)
+BATCH_ID=${OUTPUT: -36}
```

Comment:
It would be nicer to tie these steps together in the python test script rather than passing stuff around here